src/main/java/nextstep/ladder/domain/Player.java

```diff
@@ -0,0 +1,47 @@
+package nextstep.ladder.domain;
+
+import java.util.Objects;
+
+public class Player {
+    private final Name name;
+
+    private Player(final Name name) {
+        this.name = name;
+    }
+
+    public static Player of(final String name) {
+        return new Player(Name.of(name));
```

Comment:
주 생성자에게 위임할 수 있을 것 같아요.
```suggestion
        return of(Name.of(name));
```